### PR TITLE
14th Bug Crusade

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Fixed issue where first night elven, dwarven, sethrak, vrykul, qiraji emperors and kings could disappear from title history after loading savegame.
- Fixed issue where fleet model in South Sea (near Ahn'qiraj, under Kag'ram province) was shifted.
- Fixed issue where you could click "Develop Mutation" decision a few times.
- Now "Unite Faction" decision breaks alliances between tributaries who become vassals of faction title.
- Leader of the Horde and Alliance no longer can sell provinces of its vassals to adventures.
- Mortal characters prefer not to elect immortal characters (doesn't affect electors of Death God, Legion, Shath'yar religion).
- Deceitful, schemer, ambitious characters prefer not to elect immortal characters (doesn't affect electors of Death God, Legion, Shath'yar religion).
- Fixed harpies without dynasty in Bladefist (near Orgrimmar).
- Now the leader of the Cult of the Damned replaces the Lich King in Ask for Dark Gift event chain when the Lich King is defeated.
- Fixed issue where Scarlet Light character couldn't lift excommunication via Issue Declaration of Repentance decision because the Scarlet Light is heresy.
- Fixed issue where AI sacrificed prisoners and could interact with the Old Gods having non-Shath'yar religion (not being Old Gods' follower).
- Fixed issue tribal Upgrade to Castle/City/Temple didn't give buildings to upgraded holding if tribal was Death God, Legion or Shath'yar.
- Fixed issue where you could invite evil characters to participate in tournament.
- Fixed issue where the Highborne Company had no troops.
- AI no longer want to excommunicate Cenarius.
- Fixed issue where non-direct vassal of the Horde/Alliance (vassal of vassal of the Horde/Alliance) couldn't declare Independence war.

Secret changelog:
- Added new class and mana triggers to improve readability.
- Refactored event that restore Mana. Reviewers, plz test it.

### CODE REVIEW

**Comment by MemeWizard** [comment]
>24b4504

:thonk:

<details>

image
image
image

</details>

**Comment by zumbak04** [comment]
> Now both base tribal and fully upgraded tribal gives you the same buildings.
> There is no sense in upgrading them, unlike in vanilla.

It's the best way I see. Otherwise, they get nothing.

File: decisions/wc_offmap_decisions.txt [hunk 1]
```diff
@@ -248,7 +267,8 @@ offmap_decisions = {
 			}
 			FROMFROM = {
 				show_scope_change = no
-				death = { death_reason = death_sacrificed }
+				
+				death = { death_reason = death_sacrificed killer = FROM }
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply decisions/wc_offmap_decisions.txt]: Removed it since I added it only for testing.

**Comment by zumbak04** [comment]
> Alliance Leader keeps alliances

Fixed!

**Comment by geffe1** [comment]
>Fixed issue where first night elven, dwarven, sethrak, vrykul, qiraji emperors and kings could disappear from title history after loading savegame.

Appears to work properly now.
>Fixed issue where fleet model in South Sea (near Ahn'qiraj, under Kag'ram province) was shifted.

Looks just like in any other province.
>Fixed issue where you could click "Develop Mutation" decision a few times.

Seems ok.
>Now "Unite Faction" decision breaks alliances between tributaries who become vassals of faction title.

Works.
>Mortal characters prefer not to elect immortal characters (doesn't affect electors of Death God, Legion, Shath'yar religion).

Works for the Horde, still need to test something like the Scourge.

**Comment by geffe1** [comment]
>Fixed issue where the Highborne Company had no troops.

Fixed.
>AI no longer want to excommunicate Cenarius.

Appears to be fixed.
>Fixed issue where AI sacrificed prisoners and could interact with the Old Gods having non-Shath'yar religion (not being Old Gods' follower).

Ok. Dunno how to test, but sure- I'll take your word for it.
>Now the leader of the Cult of the Damned replaces the Lich King in Ask for Dark Gift event chain when the Lich King is defeated.

Yarps.

**Comment by geffe1** [comment]
>Fixed issue where non-direct vassal of the Horde/Alliance (vassal of vassal of the Horde/Alliance) couldn't declare Independence war.

Works.

>Fixed issue tribal Upgrade to Castle/City/Temple didn't give buildings to upgraded holding if tribal was Death God, Legion or Shath'yar.

Seems to appear ok? However, I think it got upgraded to a castle as soon as I committed genocide.
>Fixed issue where you could invite evil characters to participate in tournament. 

Not had it happen, at least not any evil characters that I knew were evil.
>Fixed issue where Scarlet Light character couldn't lift excommunication via Issue Declaration of Repentance decision because the Scarlet Light is heresy.

Seems ok.
>Leader of the Horde and Alliance no longer can sell provinces of its vassals to adventures.

Seems ok.

**Pull request merged**